.github/workflows/impress.yml

```diff
@@ -199,7 +201,7 @@ jobs:
       - name: Install Python
         uses: actions/setup-python@v3
         with:
-          python-version: "3.10"
+          python-version: "3.12.6"
```

Comment:
Should be a separate commit about upgrading to python 3.12? If you do this you also need to upgrade the python version in the Docker File and in the src/backend/pyproject.toml file!

Reply:
The Docker file is already in `3.12.6`, it is the test not testing on the good version:
![Capture d'écran 2024-11-18 122202](https://github.com/user-attachments/assets/07ea81c4-bdda-4bf0-aede-e0e7e502e177)

Reply:
[⬆️(CI) bump python version in backend test](https://github.com/numerique-gouv/impress/pull/425/commits/77f3b21e75e7a529bfc34afa73e3bc750131c7aa)

Reply:
And what I done is bumped the required version to 3.11 in `pyproject.toml` since `tomllib` is available ony from the 3.11.
https://github.com/numerique-gouv/impress/pull/425/commits/68e6404b2bcc6ae88d2d581b6dd359b73cbea2f8#diff-3c0d9933efa94182a3acbfeec8f5b1075f72debdee61ca34232ad6421153b8d1R26

Seems good ?